environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/response/SimpleEnvironmentResponse.java

```diff
@@ -37,6 +37,8 @@ public static Builder builder() {
 
         private boolean createFreeIpa;
 
+        private int freeIpaInstancesPerGroup;
```

Comment:
I think we should create a FreeipaResponse Object here because probably we will see more and more property here

Reply:
Done, but I am keeping the createFreeIpa in the parent class for API compatibility.